File: datasets/IR-Plag/convertai-original-java-rest-python/case-07/plagiarized/L2/01/L2.py
import numpy as np

def jumlahDiagonal(m):
    sum = 0
    for i in range(len(m)):
        sum += m[i][i]
    return sum

if __name__ == "__main__":
    m = np.array([list(map(float, input("Enter a 4 by 4 matrix row by row: ").split())) for _ in range(4)])
    print(f"Sum of the elements in the major diagonal is {jumlahDiagonal(m)}")

File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-07/plagiarized/L2/01/L2.kt
package IR

import java.util.*

object L2 {
    @JvmStatic
    fun main(args: Array<String>) {
        val sc = Scanner(System.`in`)
        print("Enter a 4 by 4 matrix row by row: ")
        val m = Array(4) { DoubleArray(4) }
        for (i in 0..3) {
            for (j in 0..3) {
                m[i][j] = sc.nextDouble()
            }
        }
        print("Sum of the elements in the major diagonal is " + jumlahDiagonal(m))
    }

    fun jumlahDiagonal(m: Array<DoubleArray>): Double {
        var sum = 0.0
        for (i in m.indices) {
            sum += m[i][i]
        }
        return sum
    }
}
